pronto/api/entry/signatures.py

```diff
@@ -252,8 +252,9 @@ def integrate_signature(e_acc, s_acc):
         }), 500
     else:
         con.commit()
+        annotation_match = check_annotation([e_acc, from_entry, s_acc], cur)
         return jsonify({
-            "status": True
+            "status": True, "data": annotation_match
```

Comment:
More informative name to the property containing the annotations:

```suggestion
            "status": True, 
            "annotations": annotation_match,
```